Nozomi/nozomi-cmc/ingest/parser.yml

```diff
@@ -0,0 +1,72 @@
+name: nozomi-cmc
+pipeline:
+  - name: parsed_event
+    external:
+      name: cef.parse-cef
+
+  - name: set_common_fields
+
+  - name: set_health_fields
+    filter: "{{ 'HEALTH' in parsed_event.message.DeviceEventClassID }}"
+
+  - name: set_audit_fields
+    filter: "{{ 'AUDIT' in parsed_event.message.DeviceEventClassID }}"
+
+  - name: set_alert_fields
+    filter: "{{ 'HEALTH' not in parsed_event.message.DeviceEventClassID and 'AUDIT' not in parsed_event.message.DeviceEventClassID }}"
+
+stages:
+  set_common_fields:
+    actions:
+      - set:
+          observer.vendor: "Nozomi"
+          observer.product: "Nozomi CMC"
+          event.module: "nozomi.cmc"
+          event.dataset: "{{ parsed_event.message.DeviceEventClassID }}"
+
+          event.outcome: "success"
+          event.kind: "event"
+          event.type: ["info"]
+          event.reason: "{{ parsed_event.message.msg }}"
+          destination.ip: "{{ parsed_event.message.dst }}"
+          destination.port: "{{ parsed_event.message.dpt }}"
+          destination.mac: "{{ parsed_event.message.dmac }}"
+          source.ip: "{{ parsed_event.message.src }}"
+          source.mac: "{{ parsed_event.message.smac }}"
+          source.port: "{{ parsed_event.message.spt }}"
+
+          # TODO: Check for audit type of event because in spec it should be `nozomi.cmc.device_version`
+          nozomi.cmc.version: "{{ parsed_event.message.DeviceVersion }}"
+
+      - set:
+          # Match only for severity of type int
+          event.severity: "{{ parsed_event.message.Severity }}"
+        filter: "{{ parsed_event.message.Severity | re_match('[1-9]|10') }}"
+
+  set_audit_fields:
+    actions:
+      - set:
+          event.category: ["session"]
+          event.dataset: "{{ parsed_event.message.DeviceEventClassID.replace(':', '.') }}"
```

Comment:
@TOUFIKIzakarya Is something expected to change the colons into dots? Is not an error in the spec?